src/main/java/io/candydoc/ddd/model/DDDPattern.java

```diff
@@ -22,5 +22,7 @@ public interface DDDPattern {
     T domainEvent();
 
     T valueObject();
+
+    T factory();
```

Comment:
As we are in DDDPattern class, it seems good not to have `domain` prefix. Do you think we should do the same for event and command ?

Reply:
We think it would be wise to keep _domainEvent_ and _domainCommand_ because that's the way we called them in the others files. Give 2 different names to the same thing may confuse people.